app/src/main/java/com/yesferal/hornsapp/app/presentation/common/custom/TextSubTextView.kt

```diff
@@ -0,0 +1,44 @@
+package com.yesferal.hornsapp.app.presentation.common.custom
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.LayoutInflater
+import android.view.View
+import androidx.constraintlayout.widget.ConstraintLayout
+import com.yesferal.hornsapp.app.R
+import kotlinx.android.synthetic.main.custom_image_text_view.view.*
+
+class TextSubTextView @JvmOverloads constructor(
+    context: Context,
+    attrs: AttributeSet? = null,
+    defStyleAttr: Int = 0
+) : ConstraintLayout(context, attrs, defStyleAttr) {
+
+    init {
+        val inflater =  context
+                .getSystemService(
+                    Context.LAYOUT_INFLATER_SERVICE
+                ) as LayoutInflater
+        inflater.inflate(R.layout.custom_text_sub_text_view, this, true)
+    }
+
+    fun setText(title: String?) {
+        title?.let {
+            titleTextView.visibility = View.VISIBLE
+            titleTextView.text = it
+        } ?: kotlin.run { visibility = View.GONE }
```

Comment:
You are hiding the whole Layout when you set a `null` value. But then you set a new (not null) value, this logic will make visible just the TextView instead of the whole layout. So you won't see the CustomView.